Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM node:0.10.36
+
+RUN mkdir -p /usr/src/app
+WORKDIR /usr/src/app
+
+COPY package.json /usr/src/app/
+RUN npm install
+COPY . /usr/src/app
+
+# replace this with your application's default port
+EXPOSE 8080 
+
+CMD [ "npm", "start" ]
```

Comment:
Thanks for the reminder, that's the reason we switched back to the base image in the first place.